core/src/main/java/bio/terra/pearl/core/model/participant/EnrolleeSourceType.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.pearl.core.model.participant;
+
+public enum EnrolleeSourceType {
+    IMPORT, // the enrollee data was imported from an external source
+    PARTICIPANT // default, they came in through standard signup flow
```

Comment:
what do you think about calling this `ENROLLMENT` instead of `PARTICIPANT`? It seems like we want this to tell us which action resulted in the creation of the enrollee, rather than who created the enrollee, does that sound that right?

Reply:
Good thought -- just talked with Whitney, who suggested `PORTAL_SITE` to reflect they came through the UI